src/components/views/right_panel/RoomHeaderButtons.tsx

```diff
@@ -218,6 +218,7 @@ export default class RoomHeaderButtons extends HeaderButtons<IProps> {
         );
         rightPanelPhaseButtons.set(RightPanelPhases.RoomSummary,
             <HeaderButton
+                key={RightPanelPhases.RoomSummary}
```

Comment:
those changes (adding the keys) are sadly unrelated to the actual PR. I was bothered by the error and did not think about what branch I am no. Should I move them to their own pr or can we just leave them here?

Reply:
this was also done in another PR now already. So it got removed from here